apps/pie-storybook/stories/pie-card-container.stories.ts

```diff
@@ -59,6 +60,26 @@ const cardContainerStoryMeta: CardContainerStoryMeta = {
                 summary: 'default',
             },
         },
+        padding: {
+            description: 'Set the padding of the card.',
+            control: {
+                type: 'text',
+            },
+            options: padding,
+            defaultValue: {
+                summary: 'none',
+            },
+        },
+        paddingX: {
+            description: 'Set the padding of the card.',
```

Comment:
maybe we can elaborate here more such as 

```suggestion
            description: 'Set the left / right padding of the card.',
```